ehrql/backends/emis.py

```diff
@@ -30,6 +32,33 @@ class EMISBackend(SQLBackend):
         ehrql.tables.beta.smoketest,
     ]
 
+    def get_emis_org_column(self):
```

Comment:
Oh I meant to say, this deserves some comments. At a minimum a link back to this PR so there's a breadcrumb trail to follow in working out what all this stuff is for. But I think a bit of inline explanation would be helpful as well.